hudi-client/src/test/java/org/apache/hudi/table/TestCleaner.java

```diff
@@ -88,10 +88,10 @@
 /**
  * Test Cleaning related logic.
  */
-public class TestCleaner extends TestHoodieClientBase {
+public class TestCleaner extends HoodieClientTestBase {
 
   private static final int BIG_BATCH_INSERT_SIZE = 500;
-  private static final Logger LOG = LogManager.getLogger(TestHoodieClientBase.class);
+  private static final Logger LOG = LogManager.getLogger(HoodieClientTestBase.class);
```

Comment:
Why not `private static final Logger LOG = LogManager.getLogger(TestCleaner.class);`?

Reply:
Yes, good catch..fixed.